src/Notify/Resources/doc/index.rst

```diff
@@ -0,0 +1,140 @@
+Symfony UX Notify
+===================
+
+Symfony UX Notify is a Symfony bundle providing light dropzones for
+file inputs in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+Symfony UX Notify is a Symfony bundle integrating realtime native
+notifications in Symfony applications using `Mercure`_.
+It is part of `the Symfony UX initiative`_.
+
+Installation
+------------
+
+Before you start, make sure you have `Symfony UX configured in your app`_.
+
+Then, install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-notify
+
+    # Don't forget to install the JavaScript dependencies as well and compile
+    $ yarn install --force
+    $ yarn encore dev
+
+Also make sure you have at least version 3.0 of
+`@symfony/stimulus-bridge`_ in your ``package.json`` file.
+
+Configuration
+-------------
+
+Symfony UX Notify can be configured to specify the Mercure hub to use:
+
+.. code-block:: yaml
+
+    # config/packages/notify.yaml
+
+    notify:
+        # Specifies the Mercure hub to use. Defaults to "mercure.hub.default"
+        mercure_hub: mercure.hub.custom
+
+
+Usage
+-----
+
+To use Symfony UX Notify you must have a `running Mercure server`_.
+
+Then, you can inject the ``NotifierInterface`` service and send messages on the ``chat/mercure`` channel::
+
+    // ...
+    use Symfony\Component\Notifier\Notification\Notification;
+    use Symfony\Component\Notifier\NotifierInterface;
+
+    class AnnounceFlashSalesCommand extends Command
+    {
+        protected static $defaultName = 'app:flash-sales:announce';
+        private $notifier;
+
+        public function __construct(NotifierInterface $notifier)
+        {
+            parent::__construct();
+
+            $this->notifier = $notifier;
+        }
+
+        protected function execute(InputInterface $input, OutputInterface $output): int
+        {
+            $this->notifier->send(new Notification('Flash sales has been started!', ['chat/mercure']));
+
+            return 0;
+        }
+    }
+
+Finally, native notifications could be displayed using the ``stream_notifications`` Twig function:
+
+.. code-block:: twig
+
+    {{ stream_notifications(['/my/topic/1', '/my/topic/2']) }}
+    {{ stream_notifications() }}
```

Comment:
I'm missing "the big picture" a little bit. Here is my understanding:

A) I send a `Notification` in PHP to the `chat/mercure` channel.
B) This communicates with https://github.com/symfony/mercure-notifier to send that notification to my Mercure hub
C) ??? How/where do the Mercure topics "connect" with this? If I listen to the `/my/topic/1` topic here... which notifications would I receive? Is the "topic" configured somewhere when sending the `Notification` to mercure?

Reply:
When you send a notification through the mercure notifier, you'll have to specify channels in the `Notification`:
```php
$this->notifier->send(new Notification('Hello native notifications!', ['chat/mercureFoo']));
```

This channel is configured in the notifier configuration, there you can set a dsn:
```yaml
framework:
    notifier:
        chatter_transports:
           mercureFoo: '%env(MERCURE_FOO_DSN)%'
```

And it that very dsn, you'll be able to configure the topics to send the notification:
```
MERCURE_FOO_DSN=mercure://default?topics[]=fooOne&topics[]=fooTwo
MERCURE_BAR_DSN=mercure://default?topics[]=bar
```

Finally, you can subscribe the topics you want using the twig helper:
```twig
{{ stream_notifications(['fooOne']) }}
```

Is it now clearer for you? Maybe it worth to be written somewhere in the documentation?

Reply:
Yes, you explained that VERY clearly. I understand that, technically, much of this should be documented in the notify + mercure bridge. But yes, it'd be great to include this in the README. I had originally thought that the `chat/mercure` channel used in your example was a special string that NEEDED to be exactly that string. Clearly, I was missing the point :)